FrontEnd/src/components/adminPage/table/UserTable.jsx

```diff
@@ -1,87 +1,74 @@
-import css from './Table.module.css';
-import {useNavigate} from 'react-router-dom';
+import { useState } from 'react';
+import css from './UserTable.module.css';
+import { useNavigate } from 'react-router-dom';
 import PaginationButtons from './PaginationButtons';
+import axios from 'axios';
+import useSWR from 'swr';
 
-const COLUMN_NAMES = ['ID', 'ФІО', 'Пошта', 'Телефон'];
-
-const USERS = [
-    {
-        id: 1,
-        name: 'Lebron',
-        surname: 'James',
-        person_email: 'user123123@user.com',
-        phone_number: '+380997777778'
-    },
-    {
-        id: 1,
-        name: 'Lebron',
-        surname: 'James',
-        person_email: 'user123123@user.com',
-        phone_number: '+380997777778'
-    },
-    {
-        id: 2222,
-        name: 'Lebron',
-        surname: 'James',
-        person_email: 'user123123@user.com',
-        phone_number: '+380997777778'
-    },
-    {
-        id: 1,
-        name: 'Lebron',
-        surname: 'James',
-        person_email: 'user123123@user.com',
-        phone_number: '-'
-    },
-    {
-        id: 1,
-        name: 'Lebron',
-        surname: 'James',
-        person_email: 'user123123@user.com',
-        phone_number: '+380997777778'
-    },
-    {
-        id: 1,
-        name: 'Lebron',
-        surname: 'James',
-        person_email: 'user123123@user.com',
-        phone_number: '+380997777778'
-    },
-
-];
+const COLUMN_NAMES = ['ID', 'ФІО', 'Пошта'];
 
 function UserTable() {
-
-    let navigate = useNavigate();
-    const routeChange = (id) =>{
-        let path = `/admin/users/${id}`;
+    const navigate = useNavigate();
+    const routeChange = (id) => {
```

Comment:
Could we select more meaningful name, like `navigateToUsers`?

Reply:
done navigate ToUsers & navigateToProfiles